### PR TITLE
Modify ports when using http scheme

### PR DESCRIPTION
**Pull request by gyfis**
The part of #14 that adds port support for the `http` scheme (cc @josegonzalez)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,12 @@ This will cause `CLICKHOUSE_URL` to be set as:
 clickhouse2://lolipop:SOME_PASSWORD@dokku-clickhouse-lolipop:9000/lolipop
 ```
 
+If you specify `CLICKHOUSE_DATABASE_SCHEME` to equal `http`, we'll also automatically adjust the URL to match the HTTP interface, e.g.
```
Comment (josegonzalez) [inline README.md:223]: This will likely fail because the readme is generated from the command file.

**Comment by gyfis** [comment]
@josegonzalez sorry about that, missed the automatic readme generator! It should be fine now

**Comment by jessedobbelaere** [comment]
Sorry to bump this merged PR, but I was curious why the HTTP url does not contain the `/$DATABASE_NAME` suffix while the clickhouse url has it, @gyfis ? E.g. plausible requires that:

```
CLICKHOUSE_DATABASE_URL='http://plausible:password@dokku-clickhouse-plausible:8123/plausible'
```

Thanks for the two PR's FYI 🙌

**Comment by gyfis** [comment]
@jessedobbelaere 
Hi! It made more sense to _not_ include the database in the URL, since you can use various CH databases on one database server (e.g. one per user in your app). Also, the [ClickHouse HTTP interface doc](https://clickhouse.com/docs/en/interfaces/http/) doesn't show this format (`URL/database`) at all, so I think it won't work at all for regular HTTP interface requests (be it ping, replica status, or queries).

**Pull request merged**